game/script-greetings.rpy

```diff
@@ -1769,7 +1769,31 @@ init 5 python:
 label greeting_night_night_owl:
     n 1unmajesu "Oh!{w=0.3} [player]!{w=1}{nw}"
     extend 3fllsslsbl " You're a night owl too,{w=0.2} huh?"
-    n 3fcsbg "N-{w=0.2}not that I have a problem with that,{w=0.2} obviously." 
+    n 3fcsbg "N-{w=0.2}not that I have a problem with that,{w=0.2} obviously."
     extend 4nchgnl " Welcome back!"
 
     return
+
+
+# Natsuki getting caught off guard by the player (again)
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._greeting_database,
+            label="natsu_startle_greeting",
+            unlocked=True,
+                affinity_range=(jn_affinity.LOVE, None),
+                ),
+            topic_group=TOPIC_TYPE_GREETING
+            )
+
+label natsu_startle_greeting:
+    n 1kcsunl "..."
+    n 1kwmunleqm "...?"
+    n 1kskemleex "A-ah!"
+    n 1kwmssleaf "[player_initial]-{w=0.1}[player]!,{w=1}{nw}"
+    extend 1uchssl " I'm so glad to see you again..."
+    n 2kslajl "Y-you scared me..{w=0.5}{nw}"
+    extend 1kwmajl " It doesn't matter. What would you like to talk about, [chosen_endearment]?"
```

Comment:
```suggestion
    n 1kcsunl "..."
    n 1kwmunleqm "...?"
    n 4uskemleex "A-{w=0.2}ah!{w=0.75}{nw}"
    $ player_initial = jn_utils.getPlayerInitial()
    extend 4ccswrlsbl " [player_initial]-{w=0.2}[player]!{w=0.5}{nw}"
    extend 2cllfllsbl " Are you {i}trying{/i} to jumpscare me or something?{w=0.75}{nw}"
    extend 2ccsemlesi " Jeez..."
    n 2clrbol "..."
    n 4clrajl "But...{w=1}{nw}"
    extend 4clrsslsbl " I'm...{w=1}{nw}"
    $ chosen_tease = jn_utils.getRandomTease()
    extend 4ccssmlsbl " glad to see you again,{w=0.2} [chosen_tease]."
    n 3fchsmleafsbl "W-{w=0.2}welcome back!"
```

Again, some good work here, but I'd suggest the above with the following feedback:

- Good to see you've used an affinity range here! Don't forget however that we don't want all content to gravitate towards players with high affinity: there's nothing particularly implicit about Nat being startled that would need her to be very close to her player - just something to keep in mind!
- As before, some missing `{w}` tags - remember, if you'd need to pause in real life to say what you're writing for Natsuki, she should pause too! For example, when she stutters at her surprise.
- As before, you've missed adding the variable for the endearment: this should always be added to ensure Nat doesn't say code that should be substituted out (if the variable didn't luckily already exist in memory), and that she will use a truly random word rather than whatever was generated last (if the variable did somehow already exist in memory).
- The main concern in this case would be the overall mood. Although this is a high affinity greeting, reading it gave more of an impression that Natsuki is vulnerable/shy: just because she is close doesn't mean she can't be sassy! I've adjusted the dialogue here to use Nat's natural defensiveness to play off how she was just startled, as it felt more in character.
- Some tags use timings that are outdated: we need to update our writing guide for this, but with our current standard we avoid using 0.1 as it is just too subtle to notice unless the player is paying close attention, and not really worth using (almost looks like a performance/stutter issue).